packages/components/src/Offer/Form/Bid.tsx

```diff
@@ -243,9 +243,7 @@ const OfferFormBid: FC<Props> = (props) => {
         <InputGroup>
           <NumberInput
             clampValueOnBlur={false}
-            min={0}
             step={Math.pow(10, -currency.decimals)}
-            precision={currency.decimals}
             allowMouseWheel
             w="full"
             onChange={(x) => setValue('bid', x)}
```

Comment:
This comment is about the next line,
why is this property format doing?
```
format={(e) => e.toString()}
```

Reply:
Removed as it doesn't seem to affect functionality